decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -28,6 +30,8 @@ def permissions
         end
         return permission_action unless permission_action.scope == :admin
 
+        allow! if user&.admin_terms_accepted? && user_has_any_role?(user, assembly, broad_check: true) && (permission_action.subject == :editor_image)
```

Comment:
Let's make it consistent with the other actions. Something like `user_can_upload_images_on_editor?` or whatever you prefer

Reply:
Done